merr/validation_err.go

```diff
@@ -0,0 +1,12 @@
+package merr
```

Comment:
Can we add errors to package `common`

Reply:
kept it as an independent package so as to allow for future addition of additional errors. also becomes a logical separation and would be easy for anyone to look at one place for all error related stuff, also so it can be accessed by linters and language servers appropiately.